docs/buildah-mkcw.1.md

```diff
@@ -0,0 +1,74 @@
+# buildah-mkcw "1" "July 2023" "buildah"
+
+## NAME
+buildah\-mkcw - Convert a conventional container image into a confidential workload image.
+
+## SYNOPSIS
+**buildah mkcw** [*options*] *source* *destination*
```

Comment:
maybe "mkcwi" instead?

Reply:
Appending an "i" seems to have been the original approach for when the verb was also applicable to containers.  I'd group this with pull/push, which have no corresponding commands which operate on containers.